src/testing/testing_base.nit

```diff
@@ -117,21 +117,32 @@ ulimit -t {{{ulimit_usertime}}} 2> /dev/null
 			end
 		end
 
-		if opt_no_color.value then
+		if not has_status then
 			if done == 0 then
 				print "* {name} ({tests.length} tests)"
 			end
 			return
 		end
 
 		line += "] {done}/{tests.length}"
-		if more_message != null then
-			line += " " + more_message
-		end
 		printn "{line}"
 	end
 
-	# Shoe the full description of the test-case.
+	# Is a status bar printed?
+	#
+	# true if color and non-verbose mode
+	fun has_status: Bool
```

Comment:
I find this name confusing considering that `show_unit_status` shows the progress bar even if `has_status == false`.